### DISCUSSION TITLE
Advice please for my website

### DISCUSSION THREAD

**Discussion by jacobkim9881** [EddieHubCommunity/support#485] [Show and tell]
Hello people. I just wanted to get some advice of my website. Your advice will be appreciated! Thanks!

https://lucid-clarke-3ee16f.netlify.app

**Comment by eddiejaoude** [comment]
Looks good. Maybe rounded corners, or that might just be my preference 😂

**Reply by jacobkim9881** [reply]
Thanks for the advice!

**Comment by Akshu-on-github** [comment]
Just a few things that I noticed...

- When the cursor clicks the place highlighted (on all of the nav elements), it doesn't trigger the link
![image](https://user-images.githubusercontent.com/61582763/101171526-e6666880-3665-11eb-9a04-ea4e974e6823.png)


- Might want to make the site responsive. Something like this happens when it's resized:
    <img src ="https://user-images.githubusercontent.com/61582763/101171745-380ef300-3666-11eb-9584-e9a3af14c484.png" height="500px" alt="image of site with reduced screen">

- I really like the "Notes" section, it's quirky! You could make the font size a bit larger, modify the links so that they aren't underlined to give each a bit more "oomph". Maybe make them into cards? That would give them emphasis as well

**Reply by jacobkim9881** [reply]
Thanks for the advice! I will try to modify based on your advice :+1: :)

**Comment by jveer634** [comment]
The site seems not mobile friendly... the icons are clustered into a group in mobile version.
![20201204_232530](https://user-images.githubusercontent.com/47923507/101197398-14f53b00-3688-11eb-9940-8ce735696422.jpg)
And one more issue is the transition isn't smooth when it scrolls up (passes the menu bar)..

**Reply by jacobkim9881** [reply]
Thanks for the advice! This advice helps alot!

**Comment by jasontwuk** [comment]
Hi, I would like to suggest you change all of the hyperlinks’ font color and hover color to make them stand out more from the background, so they are easier to read. 

![screenshot](https://user-images.githubusercontent.com/13745974/101263058-a02f0900-373a-11eb-802e-2b4495092b38.png)

**Reply by jacobkim9881** [reply]
It would look better then. Thanks for the advice!

**Comment by chryzcode** [comment]
# Firstly;
 -  ## The skills tab
 
The skills are not orderly arranged. I suggest you should have done it like this;
- Javasctipt
   - React
   - Node js
   - Express
Making the frame work on anything concerning javascript a sub under it
![jacob](https://user-images.githubusercontent.com/77321169/107209312-eaff2280-6a02-11eb-87db-57fb4a0b5f6e.jpg)


# Secondly;
- ## The about tab
 I feel there should be a kind of translation here, when I saw this I could not pick something from it or maybe translating it to a general acceptable language(English) can go a long way!
![jacob1](https://user-images.githubusercontent.com/77321169/107209957-cc4d5b80-6a03-11eb-86b9-a817c4f77cf4.jpg)

**Reply by jacobkim9881** [reply]
@chryzcodez Thanks for the advice! I appreciate your passion for Javascript. The advice is very helpful!

**Reply by chryzcode** [reply]
> @chryzcodez Thanks for the advice! I appreciate your passion for Javascript. The advice is very helpful!

Yeah, You are welcome!

**Comment by schmelto** [comment]
It looks like your page should be a single page without scrolling or only scrolling inline the boxes.

![image](https://user-images.githubusercontent.com/30869493/107232279-87d0b880-6a21-11eb-8aec-d6166e0b4237.png)

May you can remove the little scrolling
